File: internal/backend/process/testdata/python/handler.py
# Fixture handler for shim_test.go's TestPythonShim -- exercises the happy
# path and the exception path against a stub Runtime API.


def handler(event, context):
    return {"echoed": event, "requestId": context.aws_request_id, "functionName": context.function_name}


def throwing(event, context):
    raise ValueError("boom")

File: internal/backend/process/testdata/node/handler.mjs
// Fixture handler for shim_test.go's TestNodeShim — exercises the happy
// path and the exception path against a stub Runtime API.
export async function handler(event, context) {
  return { echoed: event, requestId: context.awsRequestId, functionName: context.functionName };
}

export async function throwing() {
  throw new Error('boom');
}
